conf/test_all.config

```diff
@@ -29,11 +29,14 @@ params {
 
     // Genome references
     fasta        = params.pipelines_testdata_base_path + "hum_data/reference_genome/GRCh38.s.fa.gz"
+
+    // Panel preparation
     panel        = "${projectDir}/tests/csv/panel.csv"
     phase        = true
     normalize    = true
-    compute_freq = false
+    compute_freq = true
     chunk_model  = "recursive"
+    remove_samples = "NA12878,NA12891,NA12892"
```

Comment:
I missed that one, what are those samples?

Reply:
Those samples are relative to the target samples that needs to be taken out to simulate a real case scenario.